test/run-spec-wasm2c.py

```diff
@@ -595,11 +598,18 @@ def main(args):
 
         for i, wasm_filename in enumerate(cwriter.GetModuleFilenames()):
             wasm_filename = os.path.join(out_dir, wasm_filename)
-            c_filename = utils.ChangeExt(wasm_filename, '.c')
-            args = ['-n', cwriter.GetModulePrefixUnmangled(i)]
-            wasm2c.RunWithArgs(wasm_filename, '-o', c_filename, *args)
+            c_filename_input = utils.ChangeExt(wasm_filename, '.c')
+            c_filenames = []
+            if options.num_outputs > 1:
+                for j in range(options.num_outputs):
+                    c_filenames.append(utils.ChangeExt(wasm_filename, '_' + str(j) + '.c'))
```

Comment:
How about `base = os.path.splitext(c_filename_input)[0])` and then within the loop you can do  `f'{base}_{j}.c'`.

(setting more than just the extension seems like an abuse of utils.ChangeExt).